bin/SleepRayTune.py

```diff
@@ -57,20 +57,20 @@ def SleepRayTune(config, cfg: DictConfig, num_epochs, num_gpu):
 
 @hydra.main(config_path="../config", config_name="config")
 # model=m2, dataset=m2, training=m2 in terminal
-def main(cfg: DictConfig, num_epochs=100, gpus_per_trial=1):
+def main(cfg: DictConfig, num_epochs=200, gpus_per_trial=1):
 
     logger = logging.getLogger()
 
     # restrict the number for cuda
-    os.environ["CUDA_VISIBLE_DEVICES"] = "1,4,3"
+    os.environ["CUDA_VISIBLE_DEVICES"] = "1,4,3,2"
 
     # define the parameter space
     config = {
         "enc_conv_c": tune.grid_search([10, 15, 20, 25]),
-        "enc_kern": tune.grid_search([3, 5]),
-        "enc_hidden": tune.grid_search([64, 128]),
-        "lr": tune.loguniform([1e-5, 1e-2]),
-        "weight_decay": tune.loguniform([1e-6, 1e-3]),
+        "enc_kern": tune.grid_search([3]),
+        "enc_hidden": tune.grid_search([64, 128, 192, 256]),
+        "lr": tune.loguniform(1e-5, 1e-2),
```

Comment:
When I try to run the script, I get an error message from the line above: 
```
TypeError: loguniform() missing 1 required positional argument: 'upper'
```
Are we using different versions of Ray tune?

Reply:
Did you run it on master branch?

I think I fixed it in my branch. Make sure it's `tune.loguniform(1e-5, 1e-2)` but not `tune.loguniform([1e-5, 1e-2])` in the master

Reply:
Ah that must be it. thanks